app/assets/stylesheets/partials/_modal.scss

```diff
@@ -0,0 +1,85 @@
+#modalOverlay {
+  background-color: #ffffff;
+  display: none; 
+  height: 100%;
+  left: 0;
+  margin: 0;
+  opacity: 0.7;
+  padding: 0;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 9999;
+}
+
+#modal {
+  background-color: #ffffff;
+  border: 2px solid #d9d4d9;
+  border-radius: 6px;
+  display: none;
+  left: 25%;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 20px;
+  position: absolute;
+  top: 40px;
+  width: 50%;
+  z-index: 99999;
+
+  @media (max-width: 767px) {
+    left: 15%;
+    width: 70%;
+  }
+
+  @media (max-width: 767px) {
+    left: 10%;
+    width: 80%;
+  }
+
+  h3 {
+    font-size: 15px;
+    font-weight: normal;
+    padding-bottom: 15px;
+  }
+
+  hr {
+    border: 1px solid #eef2f4;
+  }
+
+}
+
+#modalCloseButton {
+  color: #0069a0;
```

Comment:
Color literals like `#0069a0` should only be used in variable declarations; they should be referred to via variable everywhere else.